game-core/src/main/java/games/strategy/net/nio/NioWriter.java

```diff
@@ -97,13 +98,13 @@ private void loop() {
             final SocketWriteData packet = getData(channel);
             if (packet != null) {
               try {
-                if (logger.isLoggable(Level.FINEST)) {
-                  logger.finest("writing packet:" + packet + " to:" + channel.socket().getRemoteSocketAddress());
+                if (log.isLoggable(Level.FINEST)) {
+                  log.finest("writing packet:" + packet + " to:" + channel.socket().getRemoteSocketAddress());
                 }
                 final boolean done = packet.write(channel);
                 if (done) {
                   totalBytes += packet.size();
-                  if (logger.isLoggable(Level.FINE)) {
+                  if (log.isLoggable(Level.FINE)) {
```

Comment:
Here again: Why does this even exist?!
This expression should be false for almost all the time.